enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/FilterPicker.module.css

```diff
@@ -0,0 +1,11 @@
+.PickerContainer {
+  font-family: var(--mb-default-font-family);
+  font-size: 13px;
+
+  /** SDK does not have CSS resets for buttons, so we need to reset them here */
+  button[disabled],
+  button[type="button"] {
+    border: none;
+    background: none;
+  }
+}
```

Comment:
These are CSS resets to prevent browser styles from interfering. Notice the icon button's border and grey background, and the font size/family.

<img src="https://github.com/user-attachments/assets/4e7914fd-e55e-4431-8820-0e94dfe62e54" width="300">

Reply:
@deniskaber as long as the style does not leak to unwanted components, it would indeed be better to add resets to the global styles.

I noticed we have this reset in `base.styled.ts`. @npretto have you tried adding this to `PublicComponentStylesWrapper` while you were working on the style resets, and have you noticed any side effects?

```css
  /* reset button element */
  button {
    font-size: 100%;
    -webkit-appearance: none;
    border: 0;
    padding: 0;
    margin: 0;
    outline: none;
    background-color: transparent;
  }
```

Reply:
I didn't, but I think as long as we do the :where trick as here: https://github.com/metabase/metabase/blob/dbf9bfffcacad2d482c16be5fb0c80359ae6f488/enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx#L41-L55
it should be ok

Reply:
@npretto Thanks! I can see we already have both `border: 0` and `background-color: transparent` in the existing `SCOPED_CSS_RESET` reset though, I'm quite curious why it didn't apply to the `button` elements under `FilterPicker`...